run-server.sh

```diff
@@ -14,4 +14,8 @@ else
     LEVEL="warning"
 fi
 
-/usr/local/bin/gunicorn metadataproxy:app --log-level $LEVEL --workers=2 -k gevent -b $HOST:$PORT
+if [ -z "$WORKERS" ]; then
+    WORKERS="1"
```

Comment:
We should probably save that variable for the gevent concurrency. In this case the workers is the number of processes, but the concurrency would be (worker connections)[http://docs.gunicorn.org/en/stable/settings.html#worker-connections]